lib/contentful-management.ts

```diff
@@ -70,15 +70,14 @@ function createClient(
   )
 
   const adapter = createAdapter(params)
-  const makeRequestWithoutUserAgent: MakeRequestWithoutUserAgent = (options) =>
-    adapter.makeRequest({
-      ...options,
-      userAgent,
-    })
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // @ts-ignore
```

Comment:
It's not clear to me why this is here. I'd add a comment to explain (and consider - if applicable - using `@ts-expect-error` if that's something we plan to solve)